### PR TITLE
feat(LeanMessage): chapter three

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by leeyeh** [comment]
Deploy preview for *lc-docs-en* ready!

Built with commit f645aaeb8d4dcc37aad34e41b61f85d233c22bca

https://deploy-preview-55--lc-docs-en.netlify.com

File: views/realtime-guide-senior.md [hunk 1]
```diff
@@ -3,3 +3,1181 @@
...
+// Not supported yet
+```
+
+When creating `IMClient` with an `AVUser` instance that has completed the `logIn` process, the user's signature information can be directly accessed by LeanMessage from the account system of LeanStorage. This allows LeanMessage to automatically verify the client being logged in and the process of applying for signatures from the third-party server can be skipped.
+
+Once `IMClient` is logged in, all the other features work in the same way as discussed earlier.
+
+## Permission Management and Blacklisting
+
+The third-party signing mechanism helps to maintain a general security in your app, but each conversation still needs to keep its own order. For example, a chat room may need managers that can temporarily or permanently mute users that are behaving improperly. In this section, we will talk about how permission management within conversations can be implemented.
```
Comment (weakish) [inline views/realtime-guide-senior.md]: 「a general security」 security 应该是不可数名词

File: views/realtime-guide-senior.md [hunk 2]
```diff
@@ -3,3 +3,1181 @@
...
+  * @param memberId  The member's clientId
+  * @param callback  Callback function
+  */
+  public void getMemberInfo(final String memberId, final AVIMConversationMemberQueryCallback callback);
+  ```
+  ```cs
+  // Not supported yet
+  ```
+
+Each return value contains permission information of members in tuples (arrays) in the format of `<ConversationId, MemberId, ConversationMemberRole>`.
```
Comment (weakish) [inline views/realtime-guide-senior.md]: in a triple or array `<ConversationId, MemberId, ConversationMemberRole>`

File: views/realtime-guide-senior.md [hunk 3]
```diff
@@ -3,3 +3,1181 @@
...
+            }
+        }
+});
+```
+```cs
+// Pass in the name of the chat room
+tom.CreateChatRoomAsync("Chat Room");
+```
+
+When creating a chat room, you can specify its name and additional attributes (not required). The interface for creating chat rooms has the following differences comparing to [that for creating basic conversations](realtime-guide-beginner.html#creating-conversations):
```
Comment (weakish) [inline views/realtime-guide-senior.md]: 「additional attributes (not required)」 -> optional attributes

File: views/realtime-guide-senior.md [hunk 4]
```diff
@@ -3,3 +3,1181 @@
...
+After a conversation is muted, the current user will not get push notifications from it anymore. To unmute a conversation, use `Conversation#unmute`.
+
+> Tips:
+>
+> - Both chat rooms and basic conversations can be muted.
+> - `mute` and `unmute` operations will change the `mu` field in the `_Conversation` class. **Do not change the `mu` field directly in your app's dashboard**, otherwise push notifications may not work properly.
+
+### Keyword Filtering
+
+While allowing users to create group chats, you might consider filtering cuss words out from the messages sent by users. LeanMessage offers a built-in component that helps you easily implement such function. It works not only for chat rooms, but also for **basic conversations and system conversations**.
```
Comment (weakish) [inline views/realtime-guide-senior.md]: While allowing users to create group chats -> In group chats,

**Review by weakish — CHANGES_REQUESTED** [review]
中文文档关于获取聊天记录的签名部分的描述有点不太对，我在 leancloud/docs#2998 里改掉了

**Review by weakish — APPROVED** [review]

**Pull request merged**